google/datastore/v1/datastore_gapic.yaml

```diff
@@ -0,0 +1,117 @@
+type: com.google.api.codegen.ConfigProto
+generate_samples: true
+language_settings:
+  java:
+    package_name: com.google.cloud.datastore.spi.v1
+  python:
+    package_name: google.cloud.datastore.v1
+  go:
+    package_name: cloud.google.com/go/datastore/apiv1
+  csharp:
+    package_name: Google.Datastore.V1
+  ruby:
+    package_name: Google::Cloud::Datastore::V1
+  php:
+    package_name: Google\Cloud\Datastore\V1
+  nodejs:
+    package_name: "@google-cloud/datastore"
+interfaces:
+- name: google.datastore.v1.Datastore
+  collections: []
+  retry_codes_def:
+  - name: idempotent
+    retry_codes:
+    - UNAVAILABLE
+    - DEADLINE_EXCEEDED
+  - name: non_idempotent
+    retry_codes: []
+  retry_params_def:
+  - name: default
+    initial_retry_delay_millis: 100
+    retry_delay_multiplier: 1.3
+    max_retry_delay_millis: 60000
+    initial_rpc_timeout_millis: 20000
+    rpc_timeout_multiplier: 1
+    max_rpc_timeout_millis: 20000
+    total_timeout_millis: 600000
+  methods:
+  - name: Lookup
+    flattening:
+      groups:
+      - parameters:
+        - project_id
+        - read_options
+        - keys
+    required_fields:
+    - project_id
+    - read_options
+    - keys
+    request_object_method: true
+    retry_codes_name: idempotent
+    retry_params_name: default
+    timeout_millis: 60000
+  - name: RunQuery
+    required_fields:
+    - project_id
+    - partition_id
+    - read_options
+    - query
+    - gql_query
+    request_object_method: true
+    retry_codes_name: idempotent
```

Comment:
Yes, it's only for fetching data.
https://cloud.google.com/datastore/docs/concepts/queries
> A query retrieves entities from Google Cloud Datastore that meet a specified set of conditions.
Also GQL (reference: https://cloud.google.com/datastore/docs/apis/gql/gql_reference) always starts with SELECT.